app-header-layout/app-header-layout.html

```diff
@@ -91,10 +98,20 @@
 
       :host([has-scrolling-region]) > #contentContainer {
         @apply(--layout-fit);
-
         overflow-y: auto;
         -webkit-overflow-scrolling: touch;
       }
+
+      :host([fullbleed]) {
+        @apply(--layout-vertical);
+        @apply(--layout-fit);
+      }
```

Comment:
Let's move this css block to after `:host([has-scrolling-region])` so all the `:host` rules are kind of together.